### PR TITLE
add DevTask-1

### PR DESCRIPTION
**Pull request by KirillKhripach**

### CODE REVIEW

**Comment by DmitryPavlovski** [comment]
Code TOP IN THE WORLD

File: DevTask1/DevTask1/EntryPoint.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    class EntryPoint
+    {
+        /// <summary>
+        /// Entry point
+        /// </summary>
+        /// <param name="args">The command line arguments</param>
+        static void Main(string[] args)
+        {
+            try
+            {
```
Comment (IgorKheidorov) [inline DevTask1/DevTask1/EntryPoint.cs]: Check for args.Length = 0 before

File: DevTask1/DevTask1/SequencesFinderAndDisplayer.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    class SequencesFinderAndDisplayer
+    {
+        StringBuilder processedString;
+
+        /// <summary>
+        /// Constructor forms a string from array of strings
+        /// </summary>
+        /// <param name="inputString">Array of strings</param>
+        public SequencesFinderAndDisplayer(string[] inputString)
+        {
```
Comment (IgorKheidorov) [inline DevTask1/DevTask1/SequencesFinderAndDisplayer.cs]: Why do you check output twice? in main and here?

File: DevTask1/DevTask1/SequencesFinderAndDisplayer.cs [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+            {
+                sequence = new StringBuilder();
+                sequence.Append(processedString[i]);
+                for (int j = i + 1; j < processedString.Length; j++)
+                {
+                    if (processedString[j - 1] != processedString[j])
+                    {
+                        sequence.Append(processedString[j]);
+                        sequences.Add(sequence.ToString());
+                    }
```
Comment (IgorKheidorov) [inline DevTask1/DevTask1/SequencesFinderAndDisplayer.cs]: code style

File: DevTask1/DevTask1/SequencesFinderAndDisplayer.cs [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+                throw new Exception("Wrong number of arguments");
+            }
+            processedString = new StringBuilder();
+            foreach (string word in inputString)
+            {
+                processedString.Append(word);
+            }
+        }
+
+        /// <summary>
```
Comment (IgorKheidorov) [inline DevTask1/DevTask1/SequencesFinderAndDisplayer.cs]: Specify what kind of sequences dies it find

**Pull request merged**